pizza_cutter/des_pizza_cutter/_des_info.py

```diff
@@ -138,22 +123,28 @@ def get_des_y3_coadd_tile_info(*, tilename, band, campaign, medsconf):
         ii['position_offset'] = POSITION_OFFSET
 
         # psfex psf
-        ii['psf_rec'] = galsim.des.DES_PSFEx(ii['psf_path'])
-
-        # rough sky cut tests
-        ncol, nrow = ii['wcs'].get_naxis()
-        sky_bnds, ra_ccd, dec_ccd = get_rough_sky_bounds(
-            wcs=ii['wcs'],
-            position_offset=POSITION_OFFSET,
-            bounds_buffer_uv=16.0,
-            n_grid=4)
-        ii['sky_bnds'] = sky_bnds
-        ii['ra_ccd'] = ra_ccd
-        ii['dec_ccd'] = dec_ccd
-        ii['ccd_bnds'] = Bounds(0, nrow-1, 0, ncol-1)
+        ii['psfex_psf'] = galsim.des.DES_PSFEx(ii['psf_path'])
+
+        # piff.  TODO make the piff_run configurable
+        piff_data = load_piff_from_image_path(
+            image_path=ii['image_path'],
+            piff_run='y3a1-v29',
```

Comment:
I think we can pass the medsconf here? Or should we add another input?

Reply:
I think a piff_run input is the right thing to do.

Reply:
agreed

Reply:
mmk I will do this after this one is merged.